python/lsst/daf/butler/cli/butler.py

```diff
@@ -122,9 +120,17 @@ def _setupLogging(self, ctx):
         """Init the logging system and config it for the command.
 
         Subcommands may further configure the log settings."""
-        CliLog.initLog(longlog=LONG_LOG_FLAG in ctx.params)
-        if log_level_option.name() in ctx.params:
-            CliLog.setLogLevels(ctx.params[log_level_option.name()])
+        if isinstance(ctx, click.Context):
+            CliLog.initLog(longlog=ctx.params.get(long_log_option.name(), False))
+            if log_level_option.name() in ctx.params:
+                CliLog.setLogLevels(ctx.params[log_level_option.name()])
+        else:
+            # This works around a bug in sphinx-click, where it passes in the
+            # click.MultiCommand instead of the context.
+            # https://github.com/click-contrib/sphinx-click/issues/70
```

Comment:
If the fix is as easy as you imply on that issue, please file a PR. You'll probably get more traction with a PR than an issue.

Reply:
ok, I'll work on this.